app.py

```diff
@@ -299,28 +299,63 @@ def user_logged():
 
 
 # users view
-@app.route('/users', methods=['GET'])
+@app.route('/users', methods=['POST'])
 def user_view():
-    # workleft: change this method to post and firstly check admin stat
+    # Pass JWT Token in Header
     """return: detail of all registered users
     """
-    all_users = Users.query.all()
-    result = users_schema.dump(all_users)
-    return jsonify(result)
+    auth_header = request.headers.get('Authorization')
+    user_detail = user_info(auth_header)
+
+    if user_detail != 'AuthFail' and user_detail.get('admin'): # NoneType | 0| 1
+        all_users = Users.query.all()
+        result = users_schema.dump(all_users)
+        return jsonify(result)
+    
+    elif user_detail.get('admin') == 0:
+        return jsonify({
+            'Status':'Fail',
+            'Reason':'Not Admin'
+        })
+    
+    else:
+        return jsonify({
+            'Status':'Fail',
```

Comment:
Use enums for status, and add a return code

Reply:
@L04DB4L4NC3R 
I am changing them to [this](https://github.com/GDGVIT/AttendanceApp-Backend/pull/1/commits/66980e89023ee545f9a8ea8ced636abc3a7fe230).
Are they ok now?

Reply:
Works